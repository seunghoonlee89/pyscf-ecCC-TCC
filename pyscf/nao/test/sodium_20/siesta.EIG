       -2.4860
   100     1     1
    1    -3.51128    -3.48453    -3.43899    -3.37382    -3.28869    -3.18381    -3.05940    -2.91577    -2.75330    -2.57374
         -2.39829    -2.17629    -1.93380    -1.67346    -1.39633    -1.10370    -0.79802    -0.48910    -0.48910    -0.48476
         -0.46383    -0.46383    -0.42005    -0.42005    -0.35669    -0.35669    -0.27324    -0.27324    -0.17749    -0.16925
         -0.16925    -0.04384    -0.04384     0.09439     0.10409     0.10409     0.17121     0.27598     0.27598     0.47328
          0.47328     0.69440     0.69440     0.96075     0.96075     1.25854     1.25854     1.58684     1.58684     1.65801
          1.94212     1.94212     1.99485     2.31579     2.31579     2.43052     2.69153     2.69153     2.92573     3.04366
          3.04366     3.33765     3.33765     3.46255     3.53532     3.53532     4.02245     4.57132     5.04621     5.36833
          5.51532     5.55764     5.56034     5.60594     5.71595     5.92535     6.26674     6.77303     7.42671     8.15421
          8.27024     9.09539     9.14307     9.50924     9.63682    10.11085    10.15255    10.73948    11.04102    11.33713
         11.81782    12.17734    12.28284    13.66786    15.17495    16.72549    18.23690    19.61639    20.76529    21.56638
