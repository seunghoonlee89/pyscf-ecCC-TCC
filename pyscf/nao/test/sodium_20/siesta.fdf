SystemName          sodium_chain_20

XML.write .true.

DM.UseSaveDM               .true.  # .false. by default
DM.AllowReuse              .false.  # .true. by default
DM.AllowExtrapolation      .false. # .true. is default
DM.NumberPulay             5      # 0      is default
DM.MixingWeight            0.03    # 0.25   is default

MaxSCFIterations           200     # 50     is default
MeshCutoff                 120 Ry  # 100    is default
SolutionMethod             diagon  # diagon is default

PAO.SoftDefault            .true.
PAO.SplitNorm              0.17


AtomicCoordinatesFormat  Ang

NumberOfAtoms             20
NumberOfSpecies            1
 
%block ChemicalSpeciesLabel
          1          11 Na
%endblock ChemicalSpeciesLabel
 
AtomicCoordinatesFormat  Ang
AtomCoorFormatOut        Ang
%block AtomicCoordinatesAndAtomicSpecies
 -28.500000000    0.000000000    0.000000000    1
 -25.500000000    0.000000000    0.000000000    1
 -22.500000000    0.000000000    0.000000000    1
 -19.500000000    0.000000000    0.000000000    1
 -16.500000000    0.000000000    0.000000000    1
 -13.500000000    0.000000000    0.000000000    1
 -10.500000000    0.000000000    0.000000000    1
  -7.500000000    0.000000000    0.000000000    1
  -4.500000000    0.000000000    0.000000000    1
  -1.500000000    0.000000000    0.000000000    1
   1.500000000    0.000000000    0.000000000    1
   4.500000000    0.000000000    0.000000000    1
   7.500000000    0.000000000    0.000000000    1
  10.500000000    0.000000000    0.000000000    1
  13.500000000    0.000000000    0.000000000    1
  16.500000000    0.000000000    0.000000000    1
  19.500000000    0.000000000    0.000000000    1
  22.500000000    0.000000000    0.000000000    1
  25.500000000    0.000000000    0.000000000    1
  28.500000000    0.000000000    0.000000000    1
%endblock AtomicCoordinatesAndAtomicSpecies

### Molecular dynamics 
ElectronicTemperature      300 K
MD.TypeOfRun               CG
MD.NumCGsteps              0
MD.UseSaveXV              .false.      # .false. is default

COOP.Write     .true.
WriteDenchar   .true.

