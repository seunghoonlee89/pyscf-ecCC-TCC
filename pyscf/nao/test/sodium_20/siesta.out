Siesta Version: siesta-4.0b-494
Architecture  : x86_64-mkl_mpi
Compiler flags: mpif90 -I.  -ffixed-line-length-0    -g -O2
PP flags      : -DFC_HAVE_FLUSH -DFC_HAVE_ABORT -DMPI 
PARALLEL version

* Running in serial mode with MPI
>> Start of run:   4-JAN-2018  18:11:37

                           ***********************       
                           *  WELCOME TO SIESTA  *       
                           ***********************       

reinit: Reading from standard input
************************** Dump of input data file ****************************
SystemName          sodium_chain_20
XML.write .true.
DM.UseSaveDM               .true.  # .false. by default
DM.AllowReuse              .false.  # .true. by default
DM.AllowExtrapolation      .false. # .true. is default
DM.NumberPulay             5      # 0      is default
DM.MixingWeight            0.03    # 0.25   is default
MaxSCFIterations           200     # 50     is default
MeshCutoff                 120 Ry  # 100    is default
SolutionMethod             diagon  # diagon is default
PAO.SoftDefault            .true.
PAO.SplitNorm              0.17
AtomicCoordinatesFormat  Ang
NumberOfAtoms             20
NumberOfSpecies            1
%block ChemicalSpeciesLabel
          1          11 Na
%endblock ChemicalSpeciesLabel
AtomicCoordinatesFormat  Ang
AtomCoorFormatOut        Ang
%block AtomicCoordinatesAndAtomicSpecies
 -28.500000000    0.000000000    0.000000000    1
 -25.500000000    0.000000000    0.000000000    1
 -22.500000000    0.000000000    0.000000000    1
 -19.500000000    0.000000000    0.000000000    1
 -16.500000000    0.000000000    0.000000000    1
 -13.500000000    0.000000000    0.000000000    1
 -10.500000000    0.000000000    0.000000000    1
  -7.500000000    0.000000000    0.000000000    1
  -4.500000000    0.000000000    0.000000000    1
  -1.500000000    0.000000000    0.000000000    1
   1.500000000    0.000000000    0.000000000    1
   4.500000000    0.000000000    0.000000000    1
   7.500000000    0.000000000    0.000000000    1
  10.500000000    0.000000000    0.000000000    1
  13.500000000    0.000000000    0.000000000    1
  16.500000000    0.000000000    0.000000000    1
  19.500000000    0.000000000    0.000000000    1
  22.500000000    0.000000000    0.000000000    1
  25.500000000    0.000000000    0.000000000    1
  28.500000000    0.000000000    0.000000000    1
%endblock AtomicCoordinatesAndAtomicSpecies
### Molecular dynamics
ElectronicTemperature      300 K
MD.TypeOfRun               CG
MD.NumCGsteps              0
MD.UseSaveXV              .false.      # .false. is default
COOP.Write     .true.
WriteDenchar   .true.
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: sodium_chain_20
reinit: -----------------------------------------------------------------------
reinit: System Label: siesta                                                      
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
 Species number:            1  Label: Na Atomic number:          11
Ground state valence configuration:   3s01
Reading pseudopotential information in formatted form from Na.psf

Valence configuration for pseudopotential generation:
3s( 1.00) rc: 2.83
3p( 0.00) rc: 2.83
3d( 0.00) rc: 3.13
4f( 0.00) rc: 3.13
For Na, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Na                   Z=  11    Mass=  22.990        Charge= 0.17977+309
Lmxo=0 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=3
          n=1  nzeta=2  polorb=1
            splnorm:   0.17000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for Na                    (Z =  11)

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Total valence charge:    1.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  3.4111
V l=1 = -2*Zval/r beyond r=  3.4111
V l=2 = -2*Zval/r beyond r=  3.4111
All V_l potentials equal beyond r=  3.0481
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  3.4111

VLOCAL1: 99.0% of the norm of Vloc inside      4.672 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     10.647 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    3.49747
atom: Maximum radius for r*vlocal+2*Zval:    3.01027
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  3.723054   el= -0.207163   Ekb=  1.615972   kbcos=  0.232035
   l= 1   rc=  3.865328   el= -0.057123   Ekb=  0.419247   kbcos=  0.257232
   l= 2   rc=  3.963185   el=  0.002352   Ekb= -0.606425   kbcos= -0.062756

KBgen: Total number of  Kleinman-Bylander projectors:    9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 3s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.020000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    7.981102
                 energy =   -0.185382
                kinetic =    0.195082
    potential(screened) =   -0.380465
       potential(ionic) =   -0.533117

   izeta = 2
                 rmatch =    7.497538
              splitnorm =    0.170000
                 energy =   -0.159327
                kinetic =    0.268043
    potential(screened) =   -0.427370
       potential(ionic) =   -0.577521

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 3s

   izeta = 1
                     rc =    7.981102
                 energy =    0.023499
                kinetic =    0.386557
    potential(screened) =   -0.363058
       potential(ionic) =   -0.526563
atom: Total number of Sankey-type orbitals:  5

atm_pop: Valence configuration (for local Pseudopot. screening):
 3s( 1.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   7.981102
comcore: Pseudo-core radius Rcore=  4.604600
GREETS from /home/kovalp/programs/siesta/4.0-b2/Src/atom.F     2325

atom: _________________________________________________________________________

prinput: Basis input ----------------------------------------------------------
GREETS from /home/kovalp/programs/siesta/4.0-b2/Src/atom.F     7006

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   11 Na                      # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
Na                    1                    # Species label, number of l-shells
 n=3   0   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   7.981      7.498   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:    -53.85722   0.00000   0.00000  1        1
siesta:    -48.18804   0.00000   0.00000  1        2
siesta:    -42.51885   0.00000   0.00000  1        3
siesta:    -36.84967   0.00000   0.00000  1        4
siesta:    -31.18049   0.00000   0.00000  1        5
siesta:    -25.51131   0.00000   0.00000  1        6
siesta:    -19.84213   0.00000   0.00000  1        7
siesta:    -14.17295   0.00000   0.00000  1        8
siesta:     -8.50377   0.00000   0.00000  1        9
siesta:     -2.83459   0.00000   0.00000  1       10
siesta:      2.83459   0.00000   0.00000  1       11
siesta:      8.50377   0.00000   0.00000  1       12
siesta:     14.17295   0.00000   0.00000  1       13
siesta:     19.84213   0.00000   0.00000  1       14
siesta:     25.51131   0.00000   0.00000  1       15
siesta:     31.18049   0.00000   0.00000  1       16
siesta:     36.84967   0.00000   0.00000  1       17
siesta:     42.51885   0.00000   0.00000  1       18
siesta:     48.18804   0.00000   0.00000  1       19
siesta:     53.85722   0.00000   0.00000  1       20

siesta: Automatic unit cell vectors (Ang):
siesta:   71.991514    0.000000    0.000000
siesta:    0.000000    9.291514    0.000000
siesta:    0.000000    0.000000    9.291514

siesta: System type = molecule  

initatomlists: Number of atoms, orbitals, and projectors:     20   100   180

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Non-Collinear-spin run           =     F
redata: SpinPolarized (Up/Down) run      =     F
redata: Number of spin components        =     1
redata: Long output                      =     F
redata: Number of Atomic Species         =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.              =     NO
redata: Mesh Cutoff                      =   120.0000  Ry
redata: Net charge of the system         =     0.0000 |e|
redata: Min. number of SCF Iter          =        0
redata: Max. number of SCF Iter          =      200
redata: Mix DM or H after convergence    =     F
redata: Recompute H after scf cycle      =     F
redata: Performing Pulay mixing using    =     5 iterations
redata: Mix DM in first SCF step ?       =     F
redata: Write Pulay info on disk?        =     F
redata: Discard 1st Pulay DM after  kick =     F
redata: New DM Mixing Weight             =     0.0300
redata: New DM Occupancy tolerance       = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks       =     0.5000
redata: DM Tolerance for SCF             =     0.000100
redata: Require (free) Energy convergence in SCF =     F
redata: DM (free)Energy tolerance for SCF =     0.000010 eV
redata: Require Harris convergence for SCF =     F
redata: DM Harris energy tolerance for SCF =     0.000010 eV
redata: Using Saved Data (generic)   =     F
redata: Use continuation files for DM    =     T
redata: Neglect nonoverlap interactions  =     F
redata: Method of Calculation            =     Diagonalization
redata: Divide and Conquer               =     T
redata: Electronic Temperature           =     0.0019  Ry
redata: Fix the spin of the system       =     F
redata: Dynamics option                  =     Single-point calculation
redata: ***********************************************************************
Total number of electrons:    20.000000
Total ionic charge:    20.000000

* ProcessorY, Blocksize:    1  24


* Orbital distribution balance (max,min):   100   100

 Kpoints in:            1 . Kpoints trimmed:            1

siesta: k-grid: Number of k-points =     1
siesta: k-grid: Cutoff (effective) =     4.646 Ang
siesta: k-grid: Supercell and displacements
siesta: k-grid:    1   0   0      0.000
siesta: k-grid:    0   1   0      0.000
siesta: k-grid:    0   0   1      0.000

                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
       71.991514    0.000000    0.000000
        0.000000    9.291514    0.000000
        0.000000    0.000000    9.291514

outcell: Cell vector modules (Ang)   :   71.991514    9.291514    9.291514
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :   6215.1886
New_DM. Step:     1
DM re-use not allowed. Resetting always
DM.UseSaveDM  overriden !!
Initializing Density Matrix...
New grid distribution:   1
           1       1:  240    1:   32    1:   32

InitMesh: MESH =   480 x    64 x    64 =     1966080
InitMesh: (bp) =   240 x    32 x    32 =      245760
InitMesh: Mesh cutoff (required, used) =   120.000   122.863 Ry
ExtMesh (bp) on 0 =   300 x    92 x    92 =     2539200
PhiOnMesh: Number of (b)points on node 0 =               245760
PhiOnMesh: nlist on node 0 =              1302252

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -68.645014
siesta: Eions   =       159.602358
siesta: Ena     =        53.296431
siesta: Ekin    =        53.126821
siesta: Enl     =        32.106948
siesta: DEna    =         0.000004
siesta: DUscf   =         0.000000
siesta: DUext   =         0.000000
siesta: Exc     =      -133.491205
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -164.738155
siesta: Etot    =      -154.563358
siesta: FreeEng =      -154.573108

   scf: iscf   Eharris(eV)      E_KS(eV)   FreeEng(eV)    dDmax  Ef(eV)
   scf:    1     -164.7382     -154.5634     -154.5731  0.62689 -2.7291
timer: Routine,Calls,Time,% =    IterSCF     1       1.792  33.66
   scf:    2     -190.9178     -163.6242     -163.6408  1.44309 -2.5822
   scf:    3     -164.8302     -157.8090     -157.8214  0.49128 -2.6339
   scf:    4     -164.6490     -161.6842     -161.7010  0.22534 -2.4960
   scf:    5     -164.6750     -162.7444     -162.7619  0.34653 -2.4677
   scf:    6     -164.5673     -163.1179     -163.1360  0.09127 -2.4956
   scf:    7     -164.5650     -163.5216     -163.5395  0.06461 -2.4947
   scf:    8     -164.5540     -163.6760     -163.6923  0.05013 -2.4999
   scf:    9     -164.5519     -163.8621     -163.8783  0.03656 -2.4996
   scf:   10     -164.5405     -164.1339     -164.1489  0.04255 -2.4937
   scf:   11     -164.5395     -164.2379     -164.2526  0.03274 -2.4925
   scf:   12     -164.5401     -164.2931     -164.3075  0.02006 -2.4936
   scf:   13     -164.5401     -164.3402     -164.3545  0.01481 -2.4935
   scf:   14     -164.5394     -164.4472     -164.4616  0.01319 -2.4916
   scf:   15     -164.5392     -164.5630     -164.5773  0.01232 -2.4881
   scf:   16     -164.5385     -164.4954     -164.5099  0.00503 -2.4874
   scf:   17     -164.5380     -164.4999     -164.5145  0.00351 -2.4870
   scf:   18     -164.5379     -164.4989     -164.5136  0.00364 -2.4869
   scf:   19     -164.5376     -164.5355     -164.5503  0.00213 -2.4856
   scf:   20     -164.5376     -164.5414     -164.5562  0.00174 -2.4855
   scf:   21     -164.5376     -164.5377     -164.5525  0.00167 -2.4859
   scf:   22     -164.5376     -164.5405     -164.5553  0.00146 -2.4860
   scf:   23     -164.5376     -164.5387     -164.5535  0.00078 -2.4860
   scf:   24     -164.5376     -164.5405     -164.5553  0.00063 -2.4859
   scf:   25     -164.5376     -164.5404     -164.5552  0.00050 -2.4859
   scf:   26     -164.5376     -164.5394     -164.5542  0.00041 -2.4859
   scf:   27     -164.5376     -164.5377     -164.5525  0.00024 -2.4860
   scf:   28     -164.5376     -164.5396     -164.5545  0.00021 -2.4860
   scf:   29     -164.5376     -164.5391     -164.5539  0.00015 -2.4860
   scf:   30     -164.5376     -164.5396     -164.5544  0.00015 -2.4860
   scf:   31     -164.5376     -164.5395     -164.5544  0.00014 -2.4860
   scf:   32     -164.5376     -164.5393     -164.5542  0.00012 -2.4860
   scf:   33     -164.5376     -164.5386     -164.5534  0.00008 -2.4860

SCF Convergence by dMax criterion
max |DM_out - DM_in|:     0.00008175
SCF cycle converged after   33 iterations

Using DM_out to compute the final energy and forces

siesta: E_KS(eV) =             -164.5376

siesta: E_KS - E_eggbox =      -164.5376

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000    0.000000   -0.000000
----------------------------------------
   Max    0.143112
   Res    0.039645    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.143112    constrained

Stress-tensor-Voigt (kbar):       -2.52       -0.00       -0.00       -0.00        0.00        0.00
(Free)E + p*V (eV/cell)     -161.2899
Target enthalpy (eV/cell)     -164.5524
Writing WFSX for COOP/COHP in siesta.fullBZ.WFSX

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -63.155223
siesta: Eions   =       159.602358
siesta: Ena     =        53.296431
siesta: Ekin    =        46.054307
siesta: Enl     =        34.118953
siesta: DEna    =        -2.509620
siesta: DUscf   =         0.308273
siesta: DUext   =         0.000000
siesta: Exc     =      -136.203570
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -164.537583
siesta: Etot    =      -164.537583
siesta: FreeEng =      -164.552429

siesta: Final energy (eV):
siesta:  Band Struct. =     -63.155223
siesta:       Kinetic =      46.054307
siesta:       Hartree =     116.962126
siesta:    Ext. field =       0.000000
siesta:   Exch.-corr. =    -136.203570
siesta:  Ion-electron =    -239.837277
siesta:       Ion-ion =      48.486831
siesta:       Ekinion =       0.000000
siesta:         Total =    -164.537583

siesta: Atomic forces (eV/Ang):
siesta:      1   -0.088635    0.000000   -0.000000
siesta:      2   -0.143112   -0.000000   -0.000000
siesta:      3    0.093203    0.000000   -0.000000
siesta:      4   -0.060975    0.000000   -0.000000
siesta:      5    0.044741   -0.000000   -0.000000
siesta:      6   -0.036426    0.000000   -0.000000
siesta:      7    0.031420    0.000000   -0.000000
siesta:      8   -0.027864   -0.000000   -0.000000
siesta:      9    0.026226   -0.000000   -0.000000
siesta:     10   -0.025154    0.000000   -0.000000
siesta:     11    0.025111    0.000000   -0.000000
siesta:     12   -0.026259    0.000000   -0.000000
siesta:     13    0.027821    0.000000   -0.000000
siesta:     14   -0.031432   -0.000000   -0.000000
siesta:     15    0.036390    0.000000   -0.000000
siesta:     16   -0.044722   -0.000000    0.000000
siesta:     17    0.060947    0.000000   -0.000000
siesta:     18   -0.093142    0.000000   -0.000000
siesta:     19    0.143054    0.000000   -0.000000
siesta:     20    0.088807   -0.000000   -0.000000
siesta: ----------------------------------------
siesta:    Tot    0.000000    0.000000   -0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.001570   -0.000000    0.000000
siesta:    -0.000000   -0.000002    0.000000
siesta:     0.000000    0.000000   -0.000002

siesta: Cell volume =       6215.188563 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00000572          0.00000008  Ry/Bohr**3
siesta:           0.00052493          0.00000777  eV/Ang**3
siesta:           0.84104578          0.01244647  kBar
(Free)E+ p_basis*V_orbitals  =        -142.264912
(Free)Eharris+ p_basis*V_orbitals  =        -142.264912

siesta: Electric dipole (a.u.)  =   -0.002095    0.000000   -0.000000
siesta: Electric dipole (Debye) =   -0.005326    0.000000   -0.000000

timer: Elapsed wall time (sec) =      51.576
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1      62.460      62.460   100.00
Setup                  1       0.300       0.300     0.48
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1      60.756      60.756    97.27
state_init             1       0.016       0.016     0.03
hsparse                1       0.000       0.000     0.00
overlap                1       0.012       0.012     0.02
Setup_H0               1       3.216       3.216     5.15
naefs                  2       0.000       0.000     0.00
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.008       0.016     0.03
nlefsm                 2       0.032       0.064     0.10
DHSCF_Init             1       3.168       3.168     5.07
DHSCF1                 1       0.136       0.136     0.22
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       3.032       3.032     4.85
REMESH                 1       0.196       0.196     0.31
REORD                 84       0.004       0.320     0.51
PHION                  1       1.480       1.480     2.37
COMM_BSC              72       0.004       0.276     0.44
POISON                36       0.691      24.860    39.80
fft                   72       0.320      23.036    36.88
IterSCF               33       1.637      54.020    86.49
setup_H               33       1.621      53.480    85.62
DHSCF                 35       1.664      58.232    93.23
DHSCF3                35       1.609      56.320    90.17
rhoofd                35       0.464      16.232    25.99
cellXC                35       0.241       8.436    13.51
vmat                  34       0.110       3.756     6.01
writeHSX              33       0.001       0.028     0.04
compute_dm            33       0.012       0.404     0.65
diagon                33       0.012       0.400     0.64
r-eigvec              34       0.011       0.360     0.58
r-buildHS             34       0.000       0.000     0.00
rdiag                 34       0.011       0.360     0.58
rdiag1                34       0.000       0.012     0.02
rdiag2                34       0.002       0.064     0.10
rdiag3                34       0.008       0.276     0.44
rdiag4                34       0.000       0.008     0.01
r-buildD              33       0.001       0.036     0.06
MIXER                 32       0.001       0.032     0.05
WriteDM               33       0.001       0.044     0.07
PostSCF                1       3.504       3.504     5.61
DHSCF4                 1       1.912       1.912     3.06
dfscf                  1       1.580       1.580     2.53
overfsm                1       0.000       0.000     0.00
state_analysis         1       0.000       0.000     0.00
siesta_move            1       0.000       0.000     0.00
writewave              1       0.004       0.004     0.01
optical                1       0.000       0.000     0.00
  
>> End of run:   4-JAN-2018  18:12:28
